database/database_test.go

```diff
@@ -0,0 +1,133 @@
+package database
+
+import (
+	"fmt"
+	"testing"
+
+	obj "github.com/jcgallegdup/Concurrent-Document-Editor/objects"
+)
+
+const path = "../updates.db"
+const invalidID = 9001
+const ID = 1
+
+var expectedChanges = []obj.Change{obj.NewChange("insert", 0, "a"), obj.NewChange("insert", 1, "b")}
+
+func TestCreateUserReturnsIncrementedCounterReal(t *testing.T) {
+	CreateEmptyDb(path)
+	db := Database{Path: path}
+	userID, _ := db.CreateUser("TESTUSER")
+
+	if userID != ID {
+		t.Errorf("Expected userID to be 1, instead it was %d", userID)
+	}
+}
+
+func TestCreateEmptyFileReturnsIncrementedCounterReal(t *testing.T) {
```

Comment:
Maybe you should add a test that verifies CreateEmptyFile returns a revision id of 1?

Reply:
**Fixed.**